versions/3.0.4.md

```diff
@@ -249,19 +250,19 @@ In the following description, if a field is not explicitly **REQUIRED** or descr
 
 #### OpenAPI Object
 
-This is the root object of the [OpenAPI document](#openapi-description).
+This is the root object of the [OpenAPI Description](#openapi-description).
 
 ##### Fixed Fields
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="oas-version"></a>openapi | `string` | **REQUIRED**. This string MUST be the [version number](#versions) of the OpenAPI Specification that the OpenAPI document uses. The `openapi` field SHOULD be used by tooling to interpret the OpenAPI document. This is _not_ related to the API [`info.version`](#info-version) string. |
+| <a name="oas-version"></a>openapi | `string` | **REQUIRED**. This string MUST be the [version number](#versions) of the OpenAPI Specification that the OpenAPI Document uses. The `openapi` field SHOULD be used by tooling to interpret the OpenAPI Document. This is _not_ related to the API [`info.version`](#info-version) string. |
 | <a name="oas-info"></a>info | [Info Object](#info-object) | **REQUIRED**. Provides metadata about the API. The metadata MAY be used by tooling as required. |
 | <a name="oas-servers"></a>servers | [[Server Object](#server-object)] | An array of Server Objects, which provide connectivity information to a target server. If the `servers` field is not provided, or is an empty array, the default value would be a [Server Object](#server-object) with a [url](#server-url) value of `/`. |
 | <a name="oas-paths"></a>paths | [Paths Object](#paths-object) | **REQUIRED**. The available paths and operations for the API. |
-| <a name="oas-components"></a>components | [Components Object](#components-object) | An element to hold various schemas for the document. |
+| <a name="oas-components"></a>components | [Components Object](#components-object) | An element to hold various schemas for the description. |
```

Comment:
I never realized this says "to hold various schemas", which is completely inaccurate and I assume carried over from 2.0.  The correct wording would be "to hold various Objects for use in the description." (or "Description"?  "OAD"?)